zivid_camera/test/test_zivid_camera.cpp

```diff
@@ -77,6 +78,7 @@ class ZividNodeTest : public ZividNodeTestBase
   const ros::Duration medium_wait_duration{ 1.0 };
   const ros::Duration dr_get_max_wait_duration{ 5 };
   static constexpr auto capture_service_name = "/zivid_camera/capture";
+  static constexpr auto capture_service_and_save_name = "/zivid_camera/capture_and_save";
```

Comment:
this should be "capture_and_save_service_name".
It is the service name for "capture_and_save".

Reply:
done